src/Microsoft.Azure.WebJobs.ServiceBus/Triggers/ServiceBusTriggerBinding.cs

```diff
@@ -94,7 +95,8 @@ public string EntityPath
         public async Task<ITriggerData> BindAsync(object value, ValueBindingContext context)
         {
             BrokeredMessage message = value as BrokeredMessage;
-            if (message == null && !_converter.TryConvert(value, out message))
+            if (message == null && _converter != null && 
+                !_converter.TryConvert(value, out message))
```

Comment:
It'll throw elsewhere you're right though, but with a better error message hopefully